src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="inputs">
+      <form
+        class="inputs__form"
+        action="index.html"
+        data-qa="keypress"
+      >
+        <input
+          class="inputs__bigInput"
+          type="text"
+          data-qa="big"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        class="inputs__form"
+        action="index.html"
```

Comment:
Both forms have the same `action` attribute value ('index.html'). If the forms are intended to perform different actions, consider changing the `action` attribute to reflect the correct target for each form.